tensorflow_addons/optimizers/tests/gradient_accumulator_test.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for GradientAccumulator optimizers."""
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from tensorflow.python.data.experimental import AutoShardPolicy
+
+from tensorflow_addons.utils import test_utils
+from tensorflow.keras import layers
+from tensorflow_addons.optimizers import GradientAccumulator
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_run():
+    var0 = tf.Variable([1.0, 2.0])
+    var1 = tf.Variable([3.0, 4.0])
+    accum_steps = 4
+
+    grads0 = tf.constant([0.1, 0.1])
+    grads1 = tf.constant([0.01, 0.01])
+
+    grads_and_vars = list(zip([grads0, grads1], [var0, var1]))
+
+    variables = [var for _, var in grads_and_vars]
+
+    opt = GradientAccumulator(
+        tf.keras.optimizers.SGD(lr=1.0), accum_steps, trainable_variables=variables
+    )
+
+    for _ in range(accum_steps + 1):
```

Comment:
accum_steps*2+1

Reply:
You suggest to change this to:

```suggestion
    for _ in range(accum_steps * 2 + 1):
```

?